src/routes/github/create-branch/github-create-branch-post.ts

```diff
@@ -88,15 +96,15 @@ export const GithubCreateBranchPost = async (req: Request, res: Response): Promi
 		sendTrackEventAnalytics(AnalyticsTrackEventsEnum.CreateBranchSuccessTrackEventName, jiraHost);
 		const tags = {
 			name: newBranchName,
-			gitHubProduct: getCloudOrServerFromGitHubAppId(gitHubAppConfig.githubAppId)
```

Comment:
A bug.

Reply:
omg! How come this wasn't caught in a test?! @joshkay10 fyi